crates/hyperswitch_connectors/src/connectors/novalnet/transformers.rs

```diff
@@ -182,27 +182,16 @@ impl TryFrom<&NovalnetRouterData<&PaymentsAuthorizeRouterData>> for NovalnetPaym
             Some(false) | None => 0,
         };
 
-        let billing = NovalnetPaymentsRequestBilling {
-            house_no: item.router_data.get_billing_line1()?,
-            street: item.router_data.get_billing_line2()?,
-            city: Secret::new(item.router_data.get_billing_city()?),
-            zip: item.router_data.get_billing_zip()?,
-            country_code: item.router_data.get_billing_country()?,
-        };
-
-        let customer_ip = item
-            .router_data
-            .request
-            .get_browser_info()?
-            .get_ip_address()?;
-
         let customer = NovalnetPaymentsRequestCustomer {
             first_name: item.router_data.get_billing_first_name()?,
             last_name: item.router_data.get_billing_last_name()?,
             email: item.router_data.get_billing_email()?,
-            mobile: item.router_data.get_optional_billing_phone_number(),
-            billing,
-            customer_ip,
+            //for now mobile, billing, customer_ip is hardcoded to None since novalnet has limited merchant requirement, will be made dyanmic later
```

Comment:
Added minimal customer data feature. To use this the payments request body should have `"no_nc":  1`. 
With this enabled, the following fields wont be needed in request body anymore
1. billing
2. mobile
3. customer_ip